tokens/etc/0xd6dF0C579f2A65049a893fDaEC9fCE098CC19F87.json

```diff
@@ -0,0 +1,38 @@
+{
+"symbol": "UVCX",
+
+"name": "UniversalCoin X",
+
+"type": "ERC20",
+
+"address": "0xd6dF0C579f2A65049a893fDaEC9fCE098CC19F87",
```

Comment:
Hm - there does not seem to be a contract here: https://etherscan.io/address/0xd6dF0C579f2A65049a893fDaEC9fCE098CC19F87
Am I missing something?

Reply:
@ligi its on Ethereum Classic so etherscan won’t work

https://blockscout.com/etc/mainnet/address/0xd6dF0C579f2A65049a893fDaEC9fCE098CC19F87/transactions